joomla/joomla-cms
# [4.0] Further protection (requires PHP 5.6+) about an session timing attack

### Summary of Changes

This is an further protection (requires PHP 5.6+) about the session timing attack implemented in Joomla 3.8.8

This issue is now moved to the public tracker thanks for the work done on this by:
@demis-palma
@wilsonge 
@SniperSister
@mbabker
@PhilETaylor 

### Testing Instructions

- apply the patch
- login the backend
- make sure the ajax call checking for Joomla Updates still works as expected.

### Expected result

Everything works and session is aborted

### Actual result

In some very very rare cases session is not aborted.

### Documentation Changes Required

none.
